NodeProg/Infrastructure/NumberNode-TryParseInt64.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NodeProg
+{
+    public partial class NumberNode : Node
```

Comment:
Do not use the same class, it should be other class. Call it like `ParseHelpers`.